pygmt/src/triangulate.py

```diff
@@ -0,0 +1,116 @@
+"""
+triangulate - Delaunay triangulation or Voronoi partitioning and gridding of
+Cartesian data.
+"""
+import pandas as pd
+from pygmt.clib import Session
+from pygmt.helpers import (
+    GMTTempFile,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+from pygmt.io import load_dataarray
+
+
+@fmt_docstring
+@use_alias(
+    G="outgrid",
+    I="spacing",
+    J="projection",
+    R="region",
+    V="verbose",
+    b="binary",
+    d="nodata",
+    e="find",
+    f="coltypes",
+    h="header",
+    i="incols",
+    r="registration",
+)
+@kwargs_to_strings(R="sequence")
+def triangulate(table=None, x=None, y=None, z=None, **kwargs):
+    """
+    Delaunay triangulation or Voronoi partitioning and gridding of Cartesian
+    data.
+
+    Triangulate reads in x,y[,z] data and performs Delaunay triangulation,
+    i.e., it find how the points should be connected to give the most
+    equilateral triangulation possible. If a map projection (give *region*
```

Comment:
```suggestion
    Triangulate reads in x,y[,z] data and performs Delaunay triangulation
    to find how the points should be connected to create the most
    equilateral triangulation possible. If a map projection (give *region*
```

I think this reads better than using "i.e". But if the plan is to keep "i.e.", it should say "it finds" instead of "it find". I know this is lifted directly from the GMT docs; I"ll submit a PR to change that.